Resources/Textures/_DeadSpace/Objects/Weapons/Guns/Pistols/deserteagle.rsi/meta.json

```
@@ -0,0 +1,35 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
```

```suggestion
  "license": "CC-BY-SA-4.0",
```
У МК версия лицензии такая стоит, значит такую и надо